ofl/notoserifdogra/METADATA.pb

```diff
@@ -1,5 +1,5 @@
 name: "Noto Serif Dogra"
-designer: "Google"
+designer: "Ek Type"
```

Comment:
Not sure that this change is Ok on GF side

Reply:
OK. Obviously we are pulling the designer info from the font, and currently the Designer field in Noto sources is the contracting designer. I think we want to keep that in the Noto source but change to "Google" when creating the METADATA.pb.